onnxmltools/convert/main.py

```diff
@@ -18,20 +18,6 @@ def convert_coreml(model, name=None, initial_types=None, doc_string='', target_o
                    custom_conversion_functions, custom_shape_calculators)
 
 
-def convert_keras(model, name=None, initial_types=None, doc_string='',
```

Comment:
why is the keras converter wrapper removed?

Reply:
keras moved to keras-onnx?

Reply:
this is a wrapper function, just like what we did for skl2onnx

Reply:
I think we should not do that. It is confusing. It also suggests onnxmltools takes a dependency on keras-onnx and it should not.

Reply:
The decision is onnxmltools will become a portal of the converter collections. To save the cost, some small converter is still staying in this git repro.
Any concerns of not doing this?